src/components/NodeEditor/NodeEditor.tsx

```diff
@@ -596,7 +648,10 @@ export default class NodeEditor extends React.PureComponent<NodeEditorProps, Nod
                     key={'fc-back'}
                     onKeyPress={this.onKeyPress}
                     styles={formStyles.advanced}>
-                    <FormComp ref={this.formRef} {...{ ...formProps, showAdvanced: true }} />
+                    <FormComp
+                        ref={this.onFormRef}
+                        {...{ ...formProps, showAdvanced: true }}
```

Comment:
double spread 🙀

Reply:
hah not a fan? it's an easy way to overwrite props in tests but can understand if it's something you want avoid